src/test/java/com/kainos/projectdrill/resource/ViewsResourceTest.java

```diff
@@ -5,23 +5,36 @@
 import io.dropwizard.views.View;
 
 import com.kainos.projectdrill.database.JDBCConnector;
+import com.kainos.projectdrill.model.framework.Framework;
 import com.kainos.projectdrill.views.Index;
 
 import static org.junit.Assert.assertTrue;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.verify;
 
 import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
 
 public class ViewsResourceTest {
 	
 	JDBCConnector mockClient = mock(JDBCConnector.class);
+	List<Framework> frameworks =  new ArrayList<Framework>();
 	ViewsResource resource;
 	
-	// @Test
+	@Test
 	public void testSayHelloReturnsIndexView() throws SQLException, ClassNotFoundException {
```

Comment:
This is the original 'HelloWorld' test name, can you update it to something more appropriate?